packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -421,3 +444,13 @@ class NormalizedExternalError extends LoggingError {
         }
     }
 }
+
+/**
+ * Determines if a given object is an instance of a NormalizedLoggingError
+ * @param object - any object
+ * @returns - true if the object is an instance of a NormalizedLoggingError, false if not.
+ */
+export function isNormalizedLoggingError(object: unknown): object is NormalizedLoggingError {
+    return LoggingError.typeCheck(object)
+        && (object as NormalizedLoggingError).errorType === NormalizedLoggingError.normalizedErrorType;
```

Comment:
Note: If someone intentionally creates a "Generic Error", this code will wrongly consider it a "Normalized Logging Error".  e.g. [this class](https://github.com/microsoft/FluidFramework/blob/0fa1c1fc78b6629b70b86ae138597727abe42065/packages/loader/container-utils/src/error.ts#L26).

Reply:
This is why normalizeError shouldn't touch errorType and just make sure it's a LoggingError.

Reply:
I think we can get rid of this function, see my comment in DataProcessingError class